general fixes

Grekopithikos opened the pull request:
<!--
A basic changelog, goes to patch notes of the next version, so should be as short and informative as possible.
-->
## Changelog:
- Made the Light religion family view faiths and religions inside it as astray, and religions outside of it as hostile(unless if something else changes that like eastern conclave, anticleriaclism, dark fanaticism etc.
- Changed Jamieson Stormsong's name to Colm as was stated in lore.
- Renamed the Bayaka culture to Oomlot
- Renamed the Storm Cost county/barony to Storm Coast
- Made the other infusion traits glow like the undead one (fixes #574)

<!--
A changelog where you can describe more complicated things to other developers and testers.
-->
## Developer changelog:
- Moved three kingdom tier titles from the "duchy" category to the "kingdom" category in `nudger_progress.txt`

<!--
Before the merge, we recommend you to do these tests with your branch.
-->
## Tests:
- [ ] There are no errors in `wc` files in `Documents\Paradox Interactive\Crusader Kings III\logs\error.log` except `portrait_decals.cpp:101`
- [ ] The mod takes less than 5.5 GB in the Task Manager (Windows)

<!--
If you need to explain something to testers. Otherwise, delete this part.
-->
# How to test:

Grekopithikos committed:
general fixes

Grekopithikos committed:
moved fixes

Grekopithikos committed:
newtidesage

Grekopithikos committed:
religion

Grekopithikos committed:
pandaren_religion

edmund-h committed:
fix naarism hostility

edmund-h committed:
fix cecil as female name

edmund-h commented on localization/english/culture/wc_cultures_l_english.yml:
```diff
@@ -408,8 +408,8 @@
 
- pygmy:1 "Bayaka"
- pygmy_collective_noun:2 "Bayaka"
+ pygmy:1 "Pygmy"
```
please find a different name for this as this term is sometimes considered derogatory, and even worse, it's unoriginal

helloruhe replied:
this is literally in world of warcraft

edmund-h replied:
so what, it's not how things are in CK3. cultures should have names that they call themselves.  we should think of something creative. it's not immersive to play as "pygmy". dwarves don't have "dwarf" culture they have "wildhammer" and "bronzebeard" and "dark iron". "pygmy" is fine as a race name but it's just not cutting it for me as a culture name.

Grekopithikos replied:
I'll change their name to Oomlot which was the first named tribe that appeared

edmund-h committed:
fix "storm cost"

edmund-h committed:
fix value of coin event

Grekopithikos committed:
storm_coast

edmund-h committed:
properly remove character from bdp

Grekopithikos committed:
glowing infusion traits

Grekopithikos committed:
Update being_light.dds

Grekopithikos committed:
Oomlot

Grekopithikos committed:
Update 52000_ogre.txt

Grekopithikos committed:
Update wc_illustration_triggers.txt

Grekopithikos committed:
Update being_undead.dds

Grekopithikos committed:
Merge branch 'dev' into fix/code/general-fixes

Grekopithikos committed:
dragon traits look slightly better

Grekopithikos committed:
Update 53000_hillsbrad.txt

Grekopithikos committed:
Terenas Menethil II ascends to throne in -30

Grekopithikos committed:
become undead interaction icon

Grekopithikos committed:
Update 000_wc_other_titles.txt

Grekopithikos committed:
Update wc_maa_triggers.txt

FelModder commented:
- Changed Jamieson Stormsong's name to Colm as was stated in lore ✅
- Renamed the Bayaka culture to Oomlot ✅
- Renamed the Storm Cost county/barony to Storm Coast ✅
- Made the other infusion traits glow like the undead one 🟡 (working as intended and every important being trait is glowing. I'm asking a lot but please try to make Demon trait glow Orange/Red, Void Trait glow Violet, Nature Being glow Green, Arcane Being glow Blue and Light Being glow Yellow)
![being traits](https://github.com/user-attachments/assets/023c3e4f-7b84-475e-9163-3ab38a75c771)

FelModder commented:
Made the Light religion family view faiths and religions inside it as astray, and religions outside of it as hostile(unless if something else changes that like eastern conclave, anticleriaclism, dark fanaticism etc. ✅

On an unrelated note, instead of using the "Eastern Conclave" as a tenet, **Velenism** and **Lightbound** can instead use the "Eastern Conclave" as a special doctrine (like in the 3rd image)
<img width="668" alt="Velenism UI is going places" src="https://github.com/user-attachments/assets/480ae847-9afa-4544-956c-aeb56ff0cddf" />
<img width="668" alt="Lightbound UI is going places" src="https://github.com/user-attachments/assets/2c190157-e18f-4410-8319-919fe3435c17" />
<img width="350" alt="can use special doctrine instead" src="https://github.com/user-attachments/assets/e340b6f7-861c-4228-95df-ca4b6865ea26" />

FelModder committed:
dwarf daddy

edmund-h committed:
stop alliance from forming prematurely

edmund-h committed:
Merge branch 'fix/code/general-fixes' of https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2 into fix/code/general-fixes

FelModder committed:
making the eastern conclave doctrines special instead of a tenet

Grekopithikos committed:
naarism changes

Grekopithikos committed:
Merge branch 'fix/code/general-fixes' of https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2 into fix/code/general-fixes

FelModder committed:
i think naarism needs to consider eastern kingdoms astray instead of hostile

Grekopithikos committed:
Update icons.gui

FelModder committed:
Templates for different important trait icons

FelModder committed:
Lich King's trait glow gets a special color

No, if you observe neatly: The color for Lich King trait is not the same as the void being nor like the undead trait.

FelModder committed:
Special glow for all the Aspects

FelModder committed:
everything was red, oopsie daisy

FelModder committed:
final fix... hopefully

Grekopithikos committed:
Update wc_traits_l_english.yml

FelModder committed:
spaces
